python-backend/app/api/document_manager/resources/document_manager.py

```diff
@@ -83,3 +83,18 @@ def post(self):
             'document_manager_guids' : document_guid_list,
             'errors': errors,
         }
+
+    @api.doc(params={'document_guid': 'Required: Document guid. Returns the file associated to this guid.'})
+    @jwt.requires_roles(["mds-mine-create"])
+    def get(self, document_guid=None):
+
+        if document_guid == None:
+            return self.create_error_payload(401, 'Must provide a document guid.')
+
+        document_manager_doc = DocumentManager.find_by_document_manager_guid(document_guid)
+
+        if document_manager_doc:
```

Comment:
Small thing.. any reason for the difference in style for the `None` check on this line vs on like 91?

Reply:
No should be the same Ill change that now.